deliverable5/sections/installation_instruction/subsections/uninstalling.tex

```diff
@@ -0,0 +1,4 @@
+\subsection{Uninstalling the game}
+The game saves are stored in the \path{C:\Users\<your _user_name>\AppData\Roaming\Hyper} folder.
+This location can also be accessed by typing \texttt{cd "\$env:APPDATA/Hyper"} in PowerShell.
+It's advisable to remove this folder together with the game installation folder.
```

Comment:
Dont like this line. Maybe "to remove all game files from you computer remove this folder as well as the installation folder. If you want to preserve the game saves you can just deleted the installation folder.